### PR TITLE
LPS-204056 Update aria-live

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

**Comment by p2kmgcl** [comment]
Just pushed to fix SF :sweat_smile:

**Comment by p2kmgcl** [comment]
ci:test:sf

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/slider/index.html [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
 
 					<div class="carousel-caption d-md-block d-none">
-						<div data-lfr-editable-id="${index}-02-title" data-lfr-editable-type="rich-text">
+						<div id="${fragmentEntryLinkNamespace}-slide-title-${index}" data-lfr-editable-id="${index}-02-title" data-lfr-editable-type="rich-text">
 							<h2>
```
Comment (veroglez) [inline modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/slider/index.html:21]: As improvement, following the recomendations of this document https://liferay.atlassian.net/wiki/spaces/PEDS/pages/2572648693/Headings+in+DXP, can we eliminate `h2` from here?

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/slider/index.html [hunk 2]
```diff
@@ -36,14 +33,33 @@ <h2>
...
 		</div>
 
-		<button aria-controls="${fragmentEntryLinkNamespace}-carouselInner" aria-label="Previous Slide" class="carousel-control-prev" type="button">
-			<span aria-hidden="true" class="carousel-control-prev-icon"></span>
-			<span class="sr-only">Previous</span>
-		</button>
-
-		<button aria-controls="${fragmentEntryLinkNamespace}-carouselInner" aria-label="Next Slide" class="carousel-control-next" type="button">
-			<span aria-hidden="true" class="carousel-control-next-icon"></span>
+		<button aria-controls="${fragmentEntryLinkNamespace}-carouselInner" aria-label="Next Slide" class="carousel-control carousel-control-next" type="button">
```
Comment (veroglez) [inline modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/slider/index.html:36]: Can we modified the focus of the button? Right now, the screen reader takes over the entire button area, while only the icon is focused.

File: modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/slider/index.html [hunk 3]
```diff
@@ -36,14 +33,33 @@ <h2>
...
+				</span>
+				<span class="carousel-toggle-text sr-only">
+					Start slide rotation
+				</span>
+			</button>
+
+			<ol class="carousel-navigation">
+				[#list 0..configuration.numberOfSlides-1 as i]
+				<li>
+					<button aria-controls="${fragmentEntryLinkNamespace}-slide-${i+1}" aria-label="Focus slide ${i+1}" class="carousel-item-button [#if i == 0]active[/#if]" type="button">
```
Comment (veroglez) [inline modules/apps/fragment/fragment-collection-contributor/fragment-collection-contributor-basic-component/src/main/resources/com/liferay/fragment/collection/contributor/basic/component/dependencies/slider/index.html:57]: Here the action really is to select the slide. Can we change the text to "Select/Go to slide x" or something like that?

Can we also add `aria-current`? This way the user knows which is the active slide

**Comment by p2kmgcl** [comment]
@veroglez all done! :smile:

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by p2kmgcl** [comment]
ci:forward

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**